src/java.base/share/classes/java/security/Provider.java

```diff
@@ -1156,7 +1156,7 @@ public Service getService(String type, String algorithm) {
             if (s != null && (!s.isValid() || !RestrictedSecurity.isServiceAllowed(s))) {
                 legacyMap.remove(key, s);
                 // don't return invalid or disallowed legacy services
```

Comment:
Please delete line 1158 which is not present upstream.

Reply:
Removed